wicket-util/src/test/java/org/apache/wicket/util/collections/IntHashMapTest.java

```diff
@@ -48,9 +46,9 @@ public void serialize() throws IOException, ClassNotFoundException
 		ByteArrayInputStream bais = new ByteArrayInputStream(serialized);
 		ObjectInputStream ois = new ObjectInputStream(bais);
 		IntHashMap<String> deserialized = (IntHashMap<String>) ois.readObject();
-		assertThat(deserialized, is(notNullValue()));
-		assertThat(deserialized.get(1), is(equalTo("one")));
-		assertThat(deserialized.get(2), is(equalTo("two")));
+		assertNotNull(deserialized);
+		assertEquals(deserialized.get(1), "one");
```

Comment:
Is this correct parameter order?

Reply:
No you are right, it should be reversed (expected, actual)